examples/nlp_finetuning/start.py

```diff
@@ -25,8 +25,8 @@
 
     run_training_job(
         config_space=config_space,
-        mode="max",
-        metric="val_accuracy",
+        mode="min",
```

Comment:
Does this make a difference?

Reply:
yes. accuracy is no longer computed by default and since it was not added to the renate config, using it wouldn't be possible. we can keep it but then we need to add the metric.